gstd/src/benchmarks/mod.rs

```diff
@@ -0,0 +1,166 @@
+mod tests {
+    use crate::{alloc::string::ToString, prelude::*, vec, Encode, Vec};
+    use fungible_token::{contract::ActorId, ft_io::*, WASM_BINARY};
+    use gclient::{EventProcessor, GearApi, Result, WSAddress};
+
+    fn convert_i32_to_u8_array(value: i32) -> [u8; 32] {
+        let bytes = value.to_le_bytes();
+        let mut result: [u8; 32] = [0; 32];
+
+        for (i, &byte) in bytes.iter().enumerate() {
+            result[i] = byte;
+        }
+
+        result
+    }
+
+    fn convert_index_to_actor_id(index: i32) -> ActorId {
+        ActorId::from_slice(&convert_i32_to_u8_array(index)).unwrap_or_default()
+    }
+
+    const MAX_GAS_LIMIT: u64 = 250_000_000_000;
+    const BATCH_CHUNK_SIZE: usize = 25;
+
+    /// This test runs stress-loading for the fungible token contract.
+    /// Its primary purpose is to benchmark memory allocator gas consumption.
+    /// It does not verify whether the contract or the runtime works correctly.
+    ///
+    /// See [galloc optimization doc](../galloc/docs/optimization.md) for
+    /// reference.
+    #[ignore = "reason: this test requires a running node, so it is ignored by default"]
+    #[test]
+    fn stress_test_fungible_token() -> Result<()> {
+        return tokio::runtime::Builder::new_multi_thread()
```

Comment:
I think better to use #[tokio::test] instead

Reply:
Try `extend crate std` inside `mod tests`

Reply:
Already tried, didn't works

Reply:
but this works for gstd:
`#![cfg_attr(not(test), no_std)]`

but anyway it should be moved from library crate